yaml-tests/src/main/java/com/apple/foundationdb/relational/yamltests/YamlTestConfigFilters.java

```diff
@@ -77,17 +77,8 @@ boolean filter(final YamlTestConfig config) {
     CORRECT_EXPLAIN_AND_METRICS {
         @Override
         boolean filter(final YamlTestConfig config) {
-            return (boolean)config.getRunnerOptions().getOrDefault(YamlExecutionContext.OPTION_CORRECT_EXPLAIN, false) &&
-                    (boolean)config.getRunnerOptions().getOrDefault(YamlExecutionContext.OPTION_CORRECT_METRICS, false);
-        }
-    },
-    /**
-     * Used to show the plan diffs graphically.
-     */
-    SHOW_PLAN_ON_DIFF {
```

Comment:
Why are you removing `SHOW_PLAN_ON_DIFF` ?

Reply:
Sorry about that. Fixed.